adapters/eplanning/eplanning_test.go

```diff
@@ -0,0 +1,193 @@
+package eplanning
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"bytes"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"testing"
+	"time"
+)
+
+type aBidInfo struct {
+	deviceIP string
+	deviceUA string
+	tags     []aTagInfo
+	width    uint64
+	height   uint64
+	buyerUID string
+	delay    time.Duration
+}
+
+type aTagInfo struct {
+	code     string
+	bidfloor float64
+	instl    int8
+	price    float64
+	content  string
+	dealId   string
+}
+
+func TestOpenRTBRequest(t *testing.T) {
```

Comment:
Please migrate these into `.json` file tests. For examples, see the Openx PR: https://github.com/prebid/prebid-server/pull/429/files#diff-c36dcdbd80e4c5ce762a0b64ba635256

See the rationale & details here: https://github.com/prebid/prebid-server/blob/master/adapters/adapterstest/test_json.go#L17